keystore/filesystem/filesystem_backup.go

```diff
@@ -151,7 +153,10 @@ func readFilesAsKeys(files []string, basePath string, encryptor keystore2.KeyEnc
 		// remove absolute first part, leave only relative to path
 		relativeName := strings.Replace(f, basePath+"/", "", -1)
 		if isPrivate(relativeName) {
-			content, err = encryptor.Decrypt(content, getIDFromFilename(relativeName))
+			purpose, id := getIDFromFilename(relativeName)
+
+			keyContext := keystore.NewKeyContext(purpose).WithContext(id).WithZoneID(id).WithClientID(id)
```

Comment:
hm, why we initialize here context with `id` as ZoneID/Context/ClientID?

Reply:
The main reason is that we don't know about the key purpose here.

`getIDFromFilename` return purpose and ID and the purpose can also be as some zone specific or client one, so we need to pass id as Zone or ClientID and will understand what it's actually the key based on its purpose